tests/conftest/defaults.rego

```diff
@@ -0,0 +1,64 @@
+package main
+
+import data.kubernetes
+
+name := input.metadata.name
+container := input.spec.template.spec.containers[0]
+
+recommended_labels {
+  input.metadata.labels["app.kubernetes.io/managed-by"]
+}
+
+deny_deployment_name[msg] {
+  input.kind = "Deployment"
```

Comment:
```suggestion
  kubernetes.is_deployment
```

Reply:
yes, that's possible, though @corvus-ch prefers to have a little more verbosity/readability. Personally I don't have a hard feeling.

Reply:
Well, I meant to not have the package`kubernetes` at all. At least at this stage, it is not (yet) worth the complexity. I would like to prevent that the same `kubernetes.rego` file gets duplicated to other components (and then deviates into a plethora of variants). Yet I feel it is too early to already abstract a reusable library.

So my suggestion is to do the things as trivial and naive as possible for now.